e2e/app/aou-elements/textarea.ts

```diff
@@ -0,0 +1,26 @@
+import {ElementHandle, Page, WaitForSelectorOptions} from 'puppeteer';
+import TextOptions from './text-options';
+import BaseElement from './base-element';
+import {findTextarea} from './xpath-finder';
+
+export default class Textarea extends BaseElement {
+
+  constructor(aPage: Page) {
+    super(aPage);
+  }
+   
+  async withLabel(textOptions: TextOptions, waitOptions?: WaitForSelectorOptions, throwErr?: boolean): Promise<ElementHandle> {
```

Comment:
I'm confused by the pattern here. Is there a reason we want to allow someone to create a `Textarea` instance before connecting it to an actual element? i.e. should this be a static helper and `this.element` be a required constructor param?

Reply:
```
let textarea = new Textarea(this.page);
const element1 = await textarea.withLabel({normalizeSpace: FIELD_LABEL.SCIENTIFIC_APPROACHES});

In same block or function scope,  can reuse variable name textarea without creating a new instance.

const element2 = await textarea.withLabel({normalizeSpace: FIELD_LABEL.ANTICIPATED_FINDINGS});
```

Reply:
I don't think reusing the variable is helping with readability at all. This seems clearer to me:

```
const element1 = await TextArea.forLabel(this,page, {{normalizeSpace: FIELD_LABEL.SCIENTIFIC_APPROACHES});
const element2 = await TextArea.forLabel(this,page, {{normalizeSpace: FIELD_LABEL.ANTICIPATED_FINDINGS});
```